MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -1725,6 +1723,13 @@ public boolean recruitPerson(Person p, PrisonerStatus prisonerStatus, boolean gm
             }
         }
 
+        if(log){
```

Comment:
space after if and before the parenthesis to keep consistency with the rest of our codebase. Yes, I am aware this is super petty, but we want to keep things as consistent as possible to avoid having 101 different styles spread across the codebase... well... anymore than we have already :D